examples/tv-app/android/include/application-basic/ApplicationBasicManager.cpp

```diff
@@ -74,6 +74,15 @@ void ApplicationBasicManager::store(chip::EndpointId endpoint, Application * app
         ChipLogError(Zcl, "Failed to store name attribute.");
     }
 
+    MakeZclCharString(zclString, application->version);
+    EmberAfStatus versionStatus =
+        emberAfWriteServerAttribute(endpoint, ZCL_APPLICATION_BASIC_CLUSTER_ID, ZCL_APPLICATION_VERSION_ATTRIBUTE_ID,
```

Comment:
Yes. That's the plan after this PR. 

I will move all read/write functions from the example-tv-app back to the app/clusters.
Also plan to add delegate methods so you can customize your data back in example-tv-app.
I can create an issue if it helps? But it will come in a few days.

Reply:
Will be resolved under follow up. https://github.com/project-chip/connectedhomeip/issues/12853